src/config/defaults/pcbios.h

```diff
@@ -44,6 +44,7 @@ FILE_LICENCE ( GPL2_OR_LATER_OR_UBDL );
 #define	SANBOOT_PROTO_IB_SRP	/* Infiniband SCSI RDMA protocol */
 #define	SANBOOT_PROTO_FCP	/* Fibre Channel protocol */
 #define SANBOOT_PROTO_HTTP	/* HTTP SAN protocol */
+#define SANBOOT_PROTO_NBD	/* Network Block Device protocol */
```

Comment:
This should definitely not be enabled by default for BIOS builds, since they are size sensitive and NBD is a relatively obscure protocol.  (The other obscure protocol, IB_SRP, depends on the presence of an Infiniband driver and so will not be included in an Ethernet-only build)

Reply:
OK, fixed